tests/PluginTest.php

```diff
@@ -160,6 +163,31 @@ public function testGetSubscribedEvents()
         $this->assertInternalType('array', $plugin->getSubscribedEvents());
     }
 
+    public function testGetAlphabetizedList()
+    {
+        // We have to set the private method accessible
+        $method = new \ReflectionMethod(
+            '\Phergie\Irc\Plugin\React\CommandHelp\Plugin', 'alphabetize'
+        );
+        $method->setAccessible(TRUE);
+
+        //Test our method with a few plugins
+        $this->assertEquals(
+            'bofh tableflip',
```

Comment:
Instead of including two external plugins, it should be possible to implement some sort of mocking to return specific command names for this test case.